doc/migrations/3.5.0_3.5.0plus.md

```diff
@@ -0,0 +1,84 @@
+## Different MUC Light room schema definition
+
+Separate MUC Light options for the schema and default values were somewhat unintuitive and led to problems with RDBMS backend.
```

Comment:
We have introduced a change that enforces defining fields with default values.
The previous setup led to problems with the RDBMS backend as separating MUC Light options for the schema from the default values was unintuitive.
In a specific case when the default config was a subset of the schema and the client failed to provide these values when a room was created, it stored the incomplete config in the table. Then the missing config fields could not be supplied by the clients.
If you've experienced this issue, a way to fix it is described in the [Known issues](../operation-and-maintenance/known-issues.md) page.